src/app/js/formats/chart/Component.js

```diff
@@ -1,41 +1,77 @@
 import React, { Component, PropTypes } from 'react';
 import fetch from 'isomorphic-fetch';
-import { VictoryBar } from 'victory';
+import translate from 'redux-polyglot/translate';
+import { ResponsiveContainer, ReferenceLine, BarChart, Bar, XAxis, YAxis, CartesianGrid, Cell } from 'recharts';
 import { field as fieldPropTypes } from '../../propTypes';
 
-export default class ChartView extends Component {
+const moreThan = level => item => (item.value > level);
```

Comment:
Oho! You're ready to do currying, and to use Ramda. ;)

See http://developer.telerik.com/featured/practical-functional-javascript-ramda/